lib/elixir/lib/exception.ex

```diff
@@ -711,6 +711,67 @@ end
 
 defmodule ArithmeticError do
   defexception message: "bad argument in arithmetic expression"
+
+  @impl true
+  def blame(
+        %{message: default_message} = exception,
+        [{:erlang, func, args, _} | _] = stacktrace
+      ) do
+    message =
+      default_message <>
+        case {func, args} do
+          {:+, [a]} ->
+            ": tried to sum #{inspect(a)}"
+
+          {:+, [a, b]} ->
+            ": tried to sum #{inspect(a)} and #{inspect(b)}"
+
+          {:-, [a]} ->
+            ": tried to substract #{inspect(a)}"
+
+          {:-, [a, b]} ->
+            ": tried to substract #{inspect(a)} and #{inspect(b)}"
```

Comment:
Typo here: it should be "subtract" and not "substract".

Reply:
And maybe ": tried to substract #{inspect(b)} from #{inspect(a)}"?

Reply:
@fertapric I think that reads better but I think swapping the argument order will be confusing. :(